.github/workflows/deploy.yml

```diff
@@ -11,7 +11,7 @@ jobs:
    - name: Use Node.js ${{ matrix.node-version }}
      uses: actions/setup-node@v1
      with:
-       node-version: '14.x'
+       node-version: '18 .x'
```

Comment:
Quick question @jaypark25 - I'm not sure why there's an extra space here? Should it be `'18.x'` instead of `'18 .x'`?